### ISSUE TITLE
[proposal] replace dependencies with deprecated embedding

### ISSUE THREAD

**Issue by aoatmon** [GetStream/stream-chat-flutter#703]
### Please select which package this feature is related to

stream_chat_flutter

### Which platforms would this feature impact?

Android

### Is your feature request related to a problem?

given this error message
```
The plugins `image_gallery_saver, video_thumbnail` use a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if these plugins support the Android V2 embedding. Otherwise, consider removing them since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```
and these issues
https://github.com/hui-z/image_gallery_saver/issues/186
https://github.com/justsoft/video_thumbnail/issues/85

I'd consider a to adopt dependencies that use up-to-date Android embedding

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ezet** [comment]
Both of these libraries are updated with V2 embedding now, just upgrade to their latest versions.

**Comment by imtoori** [comment]
yes we'll do it in the next release

**Comment by basilmariano** [comment]
Explicitly setting **video_thumbnail: ^0.4.3** (latest version) giving version inconsistecy

Because stream_chat_flutter 3.0.0 depends on video_thumbnail ^0.3.3 and no versions of stream_chat_flutter ```
match >3.0.0 <4.0.0, stream_chat_flutter ^3.0.0 requires video_thumbnail ^0.3.3.
So, because foo depends on both stream_chat_flutter ^3.0.0 and video_thumbnail ^0.4.3, version solving failed.
pub get failed (1; So, because foo depends on both stream_chat_flutter ^3.0.0 and video_thumbnail ^0.4.3, version solving failed.)
```

What will be the solution for this?

**Comment by basilmariano** [comment]
> yes we'll do it in the next release

any expected date for the next release. I cant compile my project even running in iOS now :(

**Comment by basilmariano** [comment]
> Both of these libraries are updated with V2 embedding now, just upgrade to their latest versions.

looks like this issue in video_thumbnail is still open and sitting there without any feedback
https://github.com/justsoft/video_thumbnail/issues/85

**Comment by ezet** [comment]
The changelog says it's fixed

**Comment by xsahil03x** [comment]
Fixed in [v3.1.1](https://pub.dev/packages/stream_chat_flutter/changelog#311)

**Comment by aoatmon** [comment]
@imtoori @xsahil03x 
you sure work fast guys,  respect!
